projects/slap_manipulation/requirements.yml

```diff
@@ -0,0 +1,44 @@
+channels:
+  - conda-forge
+  - default
+  - pytorch
+dependencies:
```

Comment:
I included basic Stretch requirements into this yaml. I was thinking if someone wants to use this they can make a combined environment. However, not sure if I should put SLAP specific requirements into the main `home_robot`; not sure if the versioning would break anything.

Reply:
I think putting project-specific deps in an env file like this is fine.
We can always install them by using `conda env update`